File: Assignment47&50/program11.java
//write java program which accept number from user and return the count of even digits

import java.lang.*;
import java.util.*;

class Digit
{
    public int CountEvent(int iNo)
    {
        int idigit = 0, iCnt = 0;
        int i = 0;  

        if(iNo == -iNo)
        {
            iNo = -iNo;
        }
        while(iNo != 0)
    {
        idigit = iNo % 10;
        if(idigit % 2 == 0)
        {
            iCnt++;
        }
        iNo = iNo/10;
    }
        return iCnt;
    }
   
}


class program11
{
    public static void main(String arg[])
    {
        
        int iNo = 0;
        
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter the Number..");
        iNo = sobj.nextInt();

        Digit dobj = new Digit();
       int iRet = dobj.CountEvent(iNo);
        System.out.println("count of Even Number : "+iRet);
        
    }
}

File: Assignment5&7/program11.c
//write a program which accept number from user and return difference between summation of even digits and summation of odd digits..

#include<stdio.h>

int CountDiff(int iNo)
{
    int iDigit = 0;
    int iSum1 = 0;
    int iSum2 = 0;
    int iDiff = 0;

    while(iNo != 0)
    {
        iDigit = iNo%10;
        if((iNo % 2)==0)
        {
            iSum1 = iSum1 + iDigit;
        }

        if((iNo % 2) != 0)
        {
            iSum2 = iSum2 + iDigit;
        }
        iNo = iNo/10;
    }
    return iDiff = iSum1 - iSum2;

}



int main()
{
    int iValue = 0;
    int iRet = 0;

    printf("Enter Number\n");
    scanf("%d",&iValue);

    iRet = CountDiff(iValue);

    printf("The Difference is :%d\n",iRet);
    return 0;
}